packages/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -122,17 +129,19 @@ export const makeButtonTokens = (theme: Theme): ButtonVariantTokens => ({
       background: theme.alias.color.brand.brandBackgroundHover,
       borderColor: 'transparent',
       color: theme.alias.color.neutral.neutralForegroundInvertedAccessible,
```

Comment:
This in particular seems weird - background is using brand and foreground is using something which doesn't relate to brand. Shouldn't foregrounds and backgrounds use the same color group?

Reply:
@dzearing and I spoke offline about this and it does seem like we are missing some logic in [here](https://github.com/microsoft/fluentui/blob/4ee640f15033210d136b91e720395197a3a05adb/packages/react-theme/src/utils/light/colors.ts#L26) to add the brand foreground colors. There should also probably be a way to catch that we're not using foreground and background colors from the same alias group on any one variant that we also need to follow up with.